radio/src/targets/common/arm/stm32/stm32_usart_driver.cpp

```diff
@@ -558,6 +558,12 @@ void stm32_usart_send_byte(const stm32_usart_t* usart, uint8_t byte)
 
 void stm32_usart_send_buffer(const stm32_usart_t* usart, const uint8_t * data, uint32_t size)
 {
+
+  if (!stm32_usart_tx_completed(usart)) {
+    // Transmission is already in progress, abort
+    return;
```

Comment:
We might want to introduce return codes to be able to propagate errors. Dropping the packet here is however all we can do, except if blocking is wished, in which case we could block.